recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -360,6 +360,7 @@ def _configure_cmake(self):
         self._cmake.definitions["ENABLE_TESTING"] = False
         self._cmake.definitions["AUTORUN_UNIT_TESTS"] = False
 
+        self._cmake.definitions["BUILD_SHARED_LIBS"] = True if self.options.shared else False
```

Comment:
It's useless, it's already handled by CMake wrapper.

Reply:
Where exactly?
This solved errors I was seeing in my library when switching to Conan libraries. I didn't see the BUILD_SHARED_LIBS option being set anywhere else.

If my understanding is correct, the conanfile only sets FORCE_SHARED_CRT, which will force dynamic linking, but which doesn't force static. Because of this, if the conan shared option is set to False, and because BUILD_SHARED_LIBS is unset, it defaults to shared libraries instead of static libraries. I need static libraries.

Reply:
No, if shared is False, `BUILD_SHARED_LIBS` is set to False by CMake wrapper when invoking cmake. This has nothing to do with `FORCE_SHARED_CRT`.

Reply:
The flag is added here:
https://github.com/conan-io/conan/blob/a620e5b7ba6b0d2128edd2056b7391b5fabcd77d/conans/client/build/cmake_flags.py#L349-L353

Reply:
Thanks for the input. I must be setting that option incorrectly. I'll close this merge request and figure it out locally.